Makefile

```diff
@@ -75,9 +75,10 @@ packages/$(ARCH)/%.apk: $(KEY)
 dev-container:
 	docker run --privileged --rm -it \
 	    -v "${PWD}:${PWD}" \
+			-v "${HOME}/.cache/wolfictl/dev-container-wolfi/root:/root" \
```

Comment:
nit: should we `XDG_CACHE_HOME` this?

Reply:
Very good idea! Using `XDG_CACHE_HOME` now. I added a line a bit higher to set a default value if it's undefined.